pkg/config/config-reader.go

```diff
@@ -21,7 +21,7 @@ import (
 	"io/ioutil"
 	"os"
 
-	"github.com/pelletier/go-toml"
+	"github.com/BurntSushi/toml"
```

Comment:
Any particular reason we have changed this package. 

This package seems to be unmaintained:
https://github.com/BurntSushi/toml#this-project-is-unmaintained

Reply:
It wasn't really my choice. Maybe @dev-gaur can help here?

Reply:
tbh I don't remember when I changed to this package. @gaurav-gogia could you try reverting on `github.com/pelletier/go-toml` package and see If everthing works?

Reply:
@dev-gaur sure, will look into it